package.json

```diff
@@ -111,7 +111,7 @@
     "highlight.js": "10.4.1",
     "html-webpack-plugin": "2.30.1",
     "http-proxy-middleware": "0.17.3",
-    "jest": "23.5.0",
+    "jest": "29.7.0",
```

Comment:
@kikuomax This will probably cause an issue with all of our jest tests. Seehttps://app.circleci.com/pipelines/github/buefy/buefy/1817/workflows/bd792788-d42c-496b-b586-653d4246c7fa/jobs/3959